src/unitxt/metrics.py

```diff
@@ -567,11 +567,12 @@ def process(self, stream: Stream, stream_name: Optional[str] = None) -> Generato
 
         result = self._compute(references, predictions, task_data)
         global_score.update(self._add_score_prefixes_to_score_dict(result))
-        score_name = global_score["score_name"]
-        confidence_interval = self.compute_global_confidence_intervals(
-            references, predictions, task_data, score_name
-        )
-        global_score.update(confidence_interval)
+        score_names = self.ci_scores if self.ci_scores else [global_score["score_name"]]
```

Comment:
This is an unrelated changes of allowing ci_scores on multiple scores in global metrics.

Reply:
IS this right?